app/code/Magento/Quote/Model/ShippingAddressManagement.php

```diff
@@ -17,6 +17,8 @@
  */
 class ShippingAddressManagement implements \Magento\Quote\Model\ShippingAddressManagementInterface
 {
+    const XML_PATH_CUSTOMER_ADDRESS_COMPANY_SHOW = 'customer/address/company_show';
```

Comment:
Please, add private visibility modifier to const.

Reply:
If I change I will not be able to use it in the test class:
![Screenshot from 2021-12-04 07-29-17](https://user-images.githubusercontent.com/3149490/144706403-6624f3e6-ad7c-4069-9adf-ab968a7b5eef.png)

Reply:
You may use string in test class but implementation will be encapsulated